modules/dnn/CMakeLists.txt

```diff
@@ -227,6 +227,8 @@ if(TARGET ocv.3rdparty.openvino AND OPENCV_DNN_OPENVINO)
   endif()
 endif()
 
+set(OPENCV_DNN_BACKEND_DEFAULT "DNN_BACKEND_OPENCV" CACHE STRING "Default backend used by the DNN module")
```

Comment:
BTW, It makes sense to dump this value to CMake status too (to show through `cv::getBuildInformation()`)

Reply:
As far as we know, `cv::getBuildInformation()` prints the content of the `version_string.inc` file. What's the correct way to add further content to it? Is there a predefined CMake macro for this? Thereafter, we will fix this, too 👍

Reply:
Yes, there is a `status` macro: https://github.com/opencv/opencv/blob/114c23e41108a68a6dce5ec9ab8a900bccc47637/cmake/OpenCVUtils.cmake#L987-L993

Reply:
Thanks for pointing out 👍 
Using this macro adds a line to the `version_string.inc` file. But what's the right way to nicely integrate with the remaining formattings inside it? And where should this flag be added?

Reply:
There is no section for _dnn_ module yet, so I believe it can be added at the end of "Other libraries" section: https://github.com/opencv/opencv/blob/114c23e41108a68a6dce5ec9ab8a900bccc47637/CMakeLists.txt#L1768-L1774
Alternatively you can use `CUSTOM_STATUS` mechanism to add line to status directly from `dnn/CMakeLists.txt`: https://github.com/opencv/opencv/blob/114c23e41108a68a6dce5ec9ab8a900bccc47637/CMakeLists.txt#L1683-L1689

Reply:
Alright, thanks!
The file is simply written line by line, we thought there might be a more complex logic. We added the status after the OpenVINO status line. Most likely, this is the most related option as brining back the default-to-OpenVINO option was our main intention for this patch. Hopefully, now everything is ready for merging.